Demo/Demo/SwiftUIComponents/CardPaymentViews/CreateOrderCardPaymentView.swift

```diff
@@ -0,0 +1,73 @@
+import SwiftUI
+
+struct CreateOrderCardPaymentView: View {
```

Comment:
So it looks like we have 2 of the "relatively same" create order views. `CreateOrderCardPaymentView` and `CreateOrderPayPalWebView`. I think this is a good case where we can re-use a single view across features.
---
![Screenshot 2023-09-06 at 9 50 13 AM](https://github.com/paypal/paypal-ios/assets/35243507/ab685ef9-495b-44a2-b42e-20656287b892)
![Screenshot 2023-09-06 at 9 50 01 AM](https://github.com/paypal/paypal-ios/assets/35243507/57d65a8d-42aa-401f-8656-6684f3fdb6c6)

Reply:
I've thought about this one and having like a protocol for viewModels but the card payment has vaulting options that will be implemented with extra customer ID field and toggle button, so I thought it's better to be separate.

I think better candidate for reuse is some of the resultViews but it's a bit awkward with views spread across two PR's.
I think maybe I can do the refactoring and refinement once at least one of them get merged in.

Reply:
For vault w/ purchase (which isn't working right now) at what phase in the flow do we need the customerID? At order creation?

Reply:
Yes. In the order creation. There was a PR to create UI for card vault with purchase flow but that was before it was moved away from client side. I think there was plan for creating e2e flow for vault with purchase with order creation, so I left it in there.